example/lib/app.dart

```diff
@@ -15,8 +15,6 @@ class App extends StatelessWidget {
     return MaterialApp(
       title: title,
       home: const HomePage(title: title),
-      theme: lightTheme(),
-      darkTheme: darkTheme(),
```

Comment:
Why did you delete this?
I'd like to keep it.

Reply:
There was no reference to it in the code.

Reply:
I can’t merge with this change.

Reply:
changes made.